### PR TITLE
New package: rusty-diceware-0.2.0

### PR DESCRIPTION
**Pull request by jcgruenhage**

### CODE REVIEW

File: srcpkgs/diceware/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+short_desc="Commandline diceware, sans dice, written in rustlang"
+maintainer="jcgruenhage <jan.christian@gruenhage.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.com/yuvallanger/rusty-diceware"
+# This is not a tag in git yet, see https://gitlab.com/yuvallanger/rusty-diceware/issues/2
+distfiles="https://gitlab.com/yuvallanger/rusty-diceware/-/archive/02bf0a3e8714b936d071c7859b51c3777255c48a/rusty-diceware-02bf0a3e8714b936d071c7859b51c3777255c48a.tar.gz>diceware-v${version}.tar.gz"
+checksum=13a16c9d67f07e84af3a50616f7f81fea90958d80de14316458079ab103f1434
+
+post_install() {
+	vlicense LICENSE
```
Comment (maxice8) [inline srcpkgs/diceware/template]: don't need to include license on GPL licensed projects

Comment (jcgruenhage) [reply srcpkgs/diceware/template]: Oh, okay. So only when xlint tells me to include it?

**Comment by jcgruenhage** [comment]
Anything left to do here?

**Comment by maxice8** [comment]
Get a tagged release is the last thing to do

**Comment by jcgruenhage** [comment]
The release upstream is tagged now. I've renamed the package because the naming upstream is a bit confusing to me: The repository is called "rusty-diceware", the crate on crates.io is called "diceware". Since this is built from the repository, it shall take the name rusty-diceware IMO. Good to merge now?

**Pull request merged**